.github/workflows/release.yaml

```diff
@@ -0,0 +1,51 @@
+name: release
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
the main branch is called `master` in this repo

Reply:
Oof. Good catch.